verticapy/vdataframe.py

```diff
@@ -10046,9 +9594,9 @@ def scatter_matrix(self, columns: list = [], **style_kwds):
     def search(
```

Comment:
shouldnt there be the check_dtype decorator here>

Reply:
good one!